### DISCUSSION TITLE
v0.2.0b1

### DISCUSSION THREAD

**Discussion by sonichi** [microsoft/autogen#546] [Announcements]
This is a beta release of v0.2.0.

## Highlights
* Switching to openai v1. Please read the [migration guide](https://microsoft.github.io/autogen/docs/Installation/#migration-guide-to-v02) and report bugs.
* Support async function execution & get_human_input.
* Improvements in documentation and notebooks.

Thanks to all the reviewers for the v0.2 migration. Thanks to @aayushchhabra1999 @bonadio @marcgreen and other contributors!
 
## What's Changed
* Update Installation.md by @gagb in https://github.com/microsoft/autogen/pull/456
* Update FAQ with workaround for Issue #251 by @marcgreen in https://github.com/microsoft/autogen/pull/405
* Fix typo in README.md by @eltociear in https://github.com/microsoft/autogen/pull/481
* Fix/async function and tool execution by @aayushchhabra1999 in https://github.com/microsoft/autogen/pull/87
* Adding async support to get_human_input by @bonadio in https://github.com/microsoft/autogen/pull/466
* Added example .txt file for agentchat_langchain sample notebook by @jasondotparse in https://github.com/microsoft/autogen/pull/373
* Fix : Typo  by @AaadityaG in https://github.com/microsoft/autogen/pull/506
* Dev/v0.2 by @sonichi in https://github.com/microsoft/autogen/pull/393

## New Contributors
* @marcgreen made their first contribution in https://github.com/microsoft/autogen/pull/405
* @aayushchhabra1999 made their first contribution in https://github.com/microsoft/autogen/pull/87
* @bonadio made their first contribution in https://github.com/microsoft/autogen/pull/466
* @jasondotparse made their first contribution in https://github.com/microsoft/autogen/pull/373
* @AaadityaG made their first contribution in https://github.com/microsoft/autogen/pull/506

**Full Changelog**: https://github.com/microsoft/autogen/compare/v0.1.14...v0.2.0b1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/autogen/releases/tag/v0.2.0b1'>v0.2.0b1</a>.</em>

**Comment by kfsone** [comment]
Will this in any way make it easier to target custom OpenAI v1 API endpoints, such as presented by ollama and LM studio?

If you haven't seen LM studio, (windows approach) drop the installer into a Windows Sandbox (optional feature but available in all WIndows 10/11), install, run, point at a model, try the chat, tab on the left, try the server tab on the left, paste the example CURL command line.

ollama is basically the os-image concept of docker recognizing the LM models are .. images.

I'm personally more interested in trying GCP's Vertex models, especially bison. Using test chats via the GCP console with it produces some really exciting results for coding tasks, it blows bard out of the water in comparison. It initiates some wrong answers if you challenge it with predictable edge cases (domain-specific niches with unanswered questions on S/O or Quora) resulting in course corrections I've never seen in Bard or GP4: (paraphrasing) "Square the fox, except fox isn't a number, so lets assume I mean't the irrational number i". Feels like they perhaps trained it on transcripts of programming talks.

**Reply by sonichi** [reply]
There should be no change in the difficulty of doing that per my understanding. I could be wrong.
